Makefile.envs

```diff
@@ -115,6 +115,8 @@ export MAIN_MODULE_LDFLAGS= $(LDFLAGS_BASE) \
 	-lnodefs.js \
 	-lproxyfs.js \
 	-lworkerfs.js \
+	-lwebsocket.js \
+	-leventloop.js \
```

Comment:
For the record, these seem to add 100kb to the output size.

Reply:
I don't thing it is possible to include those as part of a side module.

Reply:
Definitely not. I think the 100kb is fine, but it's good to record it in the PR.

Reply:
I suspect most of it is from the web sockets which at some point other people will likely want to use.